### PR TITLE
Feature multi page migration guide

### PR DESCRIPTION
**Pull request by a-szotyori**
Wrote a migration & integration guide for the newest version. It's 2.6.0 now, but this version number might change as it's more a backwards compatible major update than a minor update. Also updated various documentation related files.

### Ticket 
Issue #174

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,38 +3,65 @@
...
-
-Customization of the Views is provided mostly via overriding of app resources: dimensions, strings, colors, texts, etc. Onboarding can also be customized to show your own pages, each consisting of an image and a short text.
-
-The Gini Vision Library can be used on smartphones and tablets, too. On smartphones it has been designed for portrait orientation only and will always switch to portrait orientation in both Screen API and Component API usage. On tablets both portrait and landscape orientations are supported.
-
-It is not required to limit your Activities extending the Screen API's abstract Activities or your Activities hosting the Component API's Fragments to portrait orientation. The Gini Vision Library takes care of limiting to portrait on smartphones.
+The Gini Vision Library provides Activities and Fragments for capturing, reviewing and analyzing
+photos of invoices and remittance slips.
+
+By integrating this library into your application you can allow your users to easily take a pictures
```
Comment (EarlOfEgo) [inline README.md]: take pictures

File: README.md [hunk 2]
```diff
@@ -3,38 +3,65 @@
...
+of documents, review it and - by implementing the necessary networking interfaces - upload the
```
Comment (EarlOfEgo) [inline README.md]: review them, since pictures

File: README.md [hunk 3]
```diff
@@ -3,38 +3,65 @@
...
+document to the Gini API for analysis.
+
+Communication with the Gini API is not part of the Gini Vision Library to allow clients the freedom
+of using a networking implementation of their chossing. The quickest way to add networking is to use
```
Comment (EarlOfEgo) [inline README.md]: choosing

File: README.md [hunk 4]
```diff
@@ -43,30 +70,50 @@ The Gini Vision Library consists of four main screens:
...
 
-The Component API provides four Fragments which you can include in your own layouts. This allows you more freedom to customize the Gini Vision Library, without being restricted to AppCompatActivities and the Gini Vision Library Theme.
+The Component API provides Fragments which you can include into your own layouts. This allows you
+more freedom to customize the Gini Vision Library, without being restricted to AppCompatActivities
+and the Gini Vision Library Theme.
 
-To allow usage even if you aren't using the Android Support Library, we provide a Standard and a Compat version of each Fragment.
+To allow usage even if you aren't using the Android Support Library, we have been providing a
+Standard and a Compat version of Fragments in previous versions. We still provide them for backwards
+compatibility, but for new Fragments we add we will only provide versions using the Android Support
```
Comment (EarlOfEgo) [inline README.md]: we add we will

File: README.md (hunk 3 above)
Comment (kikettas) [reply README.md]: ~to allow~, allowing

File: README.md [hunk 5]
```diff
@@ -43,30 +70,50 @@ The Gini Vision Library consists of four main screens:
 
 * Onboading: Provides useful hints to the user on how to take a perfect photo of a document.
 * Camera: The actual camera screen to capture the image of the document.
-* Review: Offers the opportunity to the user to check the sharpness of the image and to rotate it into reading direction, if necessary.
-* Analysis: Provides a UI for the analysis process of the document by showing the user a loading indicator and the image of the document.
+* Review: Offers the opportunity to the user to check the sharpness of the image and to rotate it
```
Comment (kikettas) [inline README.md]: Not sure about this section in the README, there are more component apart from these and as a developer I wouldn't find it very useful. On iOS I decided to remove it from here and added to an specific Integration guide.
What do you think? 
Also ~Onboading~ Onboarding.

File: ginivision-network/README.md [hunk 6]
```diff
@@ -1,4 +1,59 @@
...
+Gini Vision Network Library for Android
+===============================
+
+ The Gini Vision Network Library for Android provides a default implementation of the network related
+ tasks required by the Gini Vision Library.
+    
+Adding this library along with the Gini Vision Library to your application is the quickest way to
+integrate invoice scanning.
+
+In order for the Gini Vision Library to use the default implementations pass the instances to the
```
Comment (kikettas) [inline ginivision-network/README.md]: implementations **,** pass

File: ginivision/src/doc/source/updating-to-2-6-0.rst [hunk 7]
```diff
@@ -0,0 +1,635 @@
...
+    if (Intent.ACTION_VIEW.equals(action) 
+        || Intent.ACTION_SEND.equals(action)
+        || Intent.ACTION_SEND_MULTIPLE.equals(action)) {
+        ...
+    }
+
+Using the Screen API, create an Intent for launching the Gini Vision Library with
+``GiniVisionFileImport.createIntentForImportedFiles()``. This method requires a callback with which
+it will notify your app about the outcome of the import process. Since importing multiple files can
+take some seconds (images are processed and compressed) you should show an activity indicator until
```
Comment (kikettas) [inline ginivision/src/doc/source/updating-to-2-6-0.rst]: compressed),

File: README.md (hunk 5 above)
Comment (a-szotyori) [reply README.md]: I will keep it for now. It says these are the _main_ screens. We don't have an integration guide yet covering the whole library. Most information is in the Javadoc and in the update guides.

**Pull request merged**